app/models/file_set.rb

```diff
@@ -7,4 +7,9 @@ class FileSet < ActiveFedora::Base
   def to_common_index
     CommonIndexService.index(self)
   end
+
+  def processed?
+    file_set_jobs = ['CharacterizeJob', 'CreateDerivativesJob', 'CreatePyramidTiffJob', 'IngestJob']
```

Comment:
If we use the state machine, I think we are going to care about the `VisibilityCopyJob` and
`InheritPermissionsJob` as well. I know that users are going to use the UI to adjust the visibility and that will affect what appears in the index, manifest and front end.